src/CheckInput.java

```diff
@@ -53,26 +53,26 @@ public boolean isUserPinValid(String user,String pin, String outfile) throws IOE
 
         while((line = b1.readLine()) != null) {
             String[] columns = line.split("\t");
-            if (pin.equals(columns[2]))
-                userPinValid +=1;
+            if (pin.equals(columns[2].toUpperCase()))
+                pinValid = true;
 
         }
 
         f1.close();
 
-        if (userPinValid == 2) {
+        if (userValid == true && pinValid == true) {
```

Comment:
not a problem but usually you write the case that return false/ break the function first and the good return is at the end